src/assets/src/components/meetingTables.tsx

```diff
@@ -140,11 +155,22 @@ function UnstartedMeetingEditor (props: UnstartedMeetingEditorProps) {
                 </>
             );
 
+        // calculate the time different from now to the time user joined the queue
+        const [secondsInQueue, setSecondsInQueue] = useState(Math.floor((new Date().getTime() - new Date(props.meeting.created_at).getTime()) / 1000));
+
+        useEffect(() => {
+            const intervalId = setInterval(() => {
+                setSecondsInQueue(Math.floor((new Date().getTime() - new Date(props.meeting.created_at).getTime()) / 1000));
+            }, 5000);
```

Comment:
I think it looks fine going to 1000. There's not going to be any load on the server. I'm fine either way, it looks good!